Quantum-Control-Applications/Superconducting/Single Fixed Transmon/T1.py

```diff
@@ -51,52 +52,66 @@
 #####################################
 #  Open Communication with the QOP  #
 #####################################
-qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+# qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+qmm = QuantumMachinesManager()
 
-qm = qmm.open_qm(config)
+#######################
+# Simulate or execute #
+#######################
 
-job = qm.execute(T1)
-res_handles = job.result_handles
-I_handle = res_handles.get("I")
-Q_handle = res_handles.get("Q")
-iteration_handle = res_handles.get("iteration")
-I_handle.wait_for_values(1)
-Q_handle.wait_for_values(1)
-iteration_handle.wait_for_values(1)
-next_percent = 0.1  # First time print 10%
+simulate = True
```

Comment:
I would put the default to False for all of them (to make them ready to run).

Reply:
I think that for teaching purposes it is better to have it to True because this way we can show the sequence before running it

Reply:
I'm on Theo's side